assets/stylesheets/components/_inline-list.scss

```diff
@@ -0,0 +1,23 @@
+.inline-list {
+  list-style: none;
+  padding-left: 0;
+}
+
+.inline-list--item {
+  // at media "tablet or medium"
+  display: inline;
+}
+
+// at media "tablet or medium" for all of this bit
+.inline-list__slash {
+
+  .inline-list--item {
+    &:after {
+      content: "/";
+    }
+
+    &:last-child:after {
```

Comment:
At least for the time being I'd say "trailing slash is fine" - simply because the spacing wrangling for :first-child looks like being a pain 😞